server/parsec/backend/realm.py

```diff
@@ -121,32 +121,6 @@ def evolve(self, **kwargs: Any) -> RealmGrantedRole:
 
 
 class BaseRealmComponent:
-    @api
-    async def apiv2_realm_create(
-        self,
-        client_ctx: AuthenticatedClientContext,
-        req: authenticated_cmds.v2.realm_create.Req,
-    ) -> authenticated_cmds.v2.realm_create.Rep:
-        # `realm_create` command is similar between APIv2 and v4+ from the server
-        # point of view.
-        # (from client point of view, server may return `bad_timestamp` response
-        # with some fields missing)
-        # TODO: proper api req/rep conversion
-        return await self.api_realm_create(client_ctx, req)  # type: ignore[return-value, arg-type]
-
-    @api
-    async def apiv3_realm_create(
```

Comment:
APIv3 should stay !

Reply:
See https://github.com/Scille/parsec-cloud/pull/4845/files#r1276297985